schema/v1.0/parser.go

```diff
@@ -32,11 +32,11 @@ import (
 const supportedFormatMajor = 1
 
 // Maximum minor version number that this library supports.
-const supportedFormatMinor = 0
+const supportedFormatMinor = 1
```

Comment:
Doesn't this mean it should be in `schema/v1.1`?

> Each major.minor schema file format version is implemented as a separate package, with the name of the package in the vmajor.minor form.

If we aren't going to follow the readme, please remove that from the readme.

Reply:
Good catch! I am going to convert this PR to a draft to think a bit more about what's the right way forward.

Reply:
`schema/v1.1` package added. ``schema/v1.0` remains unchanged.